app/src/modelProviders/fine-tuned/getCompletion-2.ts

```diff
@@ -20,20 +20,21 @@ export async function getCompletion2(
   const prunedInput = { messages: prunedMessages, ...omit(input, "messages") };
 
   if (fineTune.baseModel === "GPT_3_5_TURBO") {
-    return getOpenaiCompletion(fineTune, prunedInput);
+    return getOpenaiCompletion(fineTune.projectId, fineTune.openaiModelId, prunedInput);
   } else {
     return getModalCompletion(fineTune, prunedInput);
   }
 }
 
-async function getOpenaiCompletion(
-  fineTune: FineTune,
+export async function getOpenaiCompletion(
+  projectId: string,
+  modelId: string | null,
```

Comment:
Can't we just get this from the `input` parameter?

Reply:
No, because sometimes we need to swap out the model that's passed in the input with another model, and reconstruct the input. We could do this swap everywhere we call getOpenaiCompletion, but I think it's cleaner to do it within the function itself.

Reply:
Let's just make the caller swap the model out when constructing the `input` object in those cases. It's really confusing to have an `input.model` parameter which is required but always ignored.

Reply:
That works for me.